src/main/java/com/dyke/poc/model/TicketComponent.java

```diff
@@ -0,0 +1,16 @@
+package com.dyke.poc.model;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
+public class TicketComponent {
```

Comment:
I added Component class before. Moreover - I am fully aware it isn't clear, but set of components are assigned to project not to user, so TicketComponent is a bit confusing namimg